tests/integration-tests/host-exports/src/mapping.ts

```diff
@@ -70,6 +70,42 @@ function testBigDecimal(): void {
 }
 
 function testEthereumAbi(): void {
+  ethereumAbiSimpleCase();
+  ethereumAbiComplexCase();
+}
+
+function ethereumAbiSimpleCase(): void {
+  let address1 = ethereum.Value.fromAddress(Address.fromString("0x0000000000000000000000000000000000000420"));
+  let address2 = ethereum.Value.fromAddress(Address.fromString("0x0000000000000000000000000000000000000680"));
+  let bool = ethereum.Value.fromBoolean(true);
+
+  let tupleArray: Array<ethereum.Value> = [
+    address2,
+    bool
+  ];
+
+  let tuple = ethereum.Value.fromTuple(tupleArray as ethereum.Tuple);
+
+  let params: Array<ethereum.Value> = [
+    address1,
+    tuple
+  ];
+
+  let encoded = ethereum.encode(params)!;
+
+  let decoded = ethereum.decode("(address,(address,bool))", encoded).toTuple();
```

Comment:
The simple case I meant was not a tuple at all, but just `.decode("address", ..`

Reply:
Oh okay, fixing!